operations/workplace/people/working-at-mattermost/README.md

```diff
@@ -4,13 +4,13 @@ description: A guide for Mattermost staff working anywhere
 
 # Working at Mattermost
 
-### **Work Anywhere**
+### **Work Across the Globe**
```

Comment:
I just reas the title and the first thing that came to mind was “Would I have to work at more than one place?”

Just a thought from a non-native english speaker.

Reply:
Great feedback. I'll ask this in the R&D leads channel to see how people react.